pkg/kubelet/kubelet_node_status_test.go

```diff
@@ -194,12 +194,12 @@ func TestUpdateNewNodeStatus(t *testing.T) {
 				ContainerManager: cm.NewStubContainerManager(),
 				allocatableReservation: v1.ResourceList{
 					v1.ResourceCPU:              *resource.NewMilliQuantity(200, resource.DecimalSI),
-					v1.ResourceMemory:           *resource.NewQuantity(100E6, resource.BinarySI),
+					v1.ResourceMemory:           *resource.NewQuantity(100e6, resource.BinarySI),
```

Comment:
hooray for gofmt

EDIT: Actually, it looks like this file has no changes. Mind dropping it from the PR?